package.json

```
@@ -18,10 +19,12 @@
   },
   "devDependencies": {
     "@hapi/code": "8.x.x",
-    "@hapi/lab": "24.x.x"
+    "@hapi/lab": "24.x.x",
```

This minimum version is necessary because it contains the fix in https://github.com/hapijs/lab/pull/1007 for functionality used by the test suite here.
```suggestion
    "@hapi/lab": "^24.2.1",
```